library/src/main/java/bio/terra/stairway/Control.java

```diff
@@ -0,0 +1,583 @@
+package bio.terra.stairway;
+
+import bio.terra.stairway.exception.DatabaseOperationException;
+import bio.terra.stairway.exception.FlightException;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import javax.sql.DataSource;
+
+@SuppressFBWarnings(
+    value = "RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE",
+    justification = "Spotbugs doesn't understand resource try construct")
+public class Control {
+  private final Stairway stairway;
+  private final DataSource dataSource;
+  private final FlightDao flightDao;
+  private final StairwayInstanceDao stairwayInstanceDao;
+
+  Control(
+      Stairway stairway,
+      DataSource dataSource,
+      FlightDao flightDao,
+      StairwayInstanceDao stairwayInstanceDao) {
+    this.stairway = stairway;
+    this.dataSource = dataSource;
+    this.flightDao = flightDao;
+    this.stairwayInstanceDao = stairwayInstanceDao;
+  }
+
+  // -- Flight methods --
+
+  private static final String flightSelectFrom =
```

Comment:
These constant names don't follow the google style guidelines. Is there a reason to not follow the guidelines here?

https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names